osu.Game/Screens/Play/SaveFailedScoreButton.cs

```diff
@@ -60,11 +60,16 @@ private void load(OsuGame? game, Player? player)
 
                         case DownloadState.NotDownloaded:
                             state.Value = DownloadState.Importing;
-                            Task.Run(importFailedScore).ContinueWith(t =>
+
+                            if (importFailedScore != null)
```

Comment:
Not suggesting anything here, but I was confused as to how this turned to become nullable but it appears it's because `FailOverlay.SaveReplay` is also nullable. I would've changed `FailOverlay` to guarantee action is not null before instantiating `SaveFailedScoreButton`, but either or is fine at this point.